modules/iscsi/udiskslinuxmanageriscsiinitiator.c

```diff
@@ -657,6 +724,34 @@ handle_login (UDisksManagerISCSIInitiator *object,
       goto out;
     }
 
+  /* sit and wait until the device appears on dbus */
+  if (udisks_daemon_wait_for_object_sync (manager->daemon,
+                                          wait_for_iscsi_object,
+                                          g_strdup (arg_name),
```

Comment:
Something has to free this duplicate string.

Reply:
`g_free` below should do that (4th parameter is free function for "user data")

Reply:
Oh, okay then. Great! :)